components/nuclio/delete/component.yaml

```diff
@@ -0,0 +1,13 @@
+name: nuclio delete
+description: delete a nuclio function.
+inputs:
+  - {name: Name, type: String, description: 'function name'}
+  - {name: Namespace, type: String, description: 'Kubernetes namespace', default: ''}
```

Comment:
I do not see the Namespace being passed to the program.

Reply:
@Ark-kun i will add

Reply:
@Ark-kun BTW is there a way to pass options w/o values ?  e.g. we have `-v` option for verbose, it doesn't accept a value, just will turn on verbose if the flag exists. if i add it now it will always add `-v` to the call and i rather not change the executable to accept `-v=true`